File: specs/query_dsl/full_text/simple_query_string/simple_query_string_flags.ts

/**namespace:QueryDsl.FullText.SimpleQueryString */
enum SimpleQueryStringFlags {
	NONE = 1,
	AND = 2,
	OR = 4,
	NOT = 8,
	PREFIX = 16,
	PHRASE = 32,
	PRECEDENCE = 64,
	ESCAPE = 128,
	WHITESPACE = 256,
	FUZZY = 512,
	NEAR = 1024,
	SLOP = 2048,
	ALL = 4096
}

File: output/csharp/QueryDsl/FullText/SimpleQueryString/SimpleQueryStringFlags.cs

using System.Runtime.Serialization;

namespace Nest.QueryDsl {

	public enum SimpleQueryStringFlags {
  
		[DataMember(Name = "NONE")] None,
		[DataMember(Name = "AND")] And,
		[DataMember(Name = "OR")] Or,
		[DataMember(Name = "NOT")] Not,
		[DataMember(Name = "PREFIX")] Prefix,
		[DataMember(Name = "PHRASE")] Phrase,
		[DataMember(Name = "PRECEDENCE")] Precedence,
		[DataMember(Name = "ESCAPE")] Escape,
		[DataMember(Name = "WHITESPACE")] Whitespace,
		[DataMember(Name = "FUZZY")] Fuzzy,
		[DataMember(Name = "NEAR")] Near,
		[DataMember(Name = "SLOP")] Slop,
		[DataMember(Name = "ALL")] All
	}
}
